Core/Steps/ReleaseProcessStepBase.cs

```diff
@@ -93,16 +93,4 @@ protected void CreateTagWithMessage (string tagName)
   {
     GitClient.Tag(tagName, $"Create tag with version {tagName}");
   }
-
-  protected SemanticVersion? CreateNewSupportBranch (SemanticVersion nextVersion)
```

Comment:
This code was already unnused? Because I can't find a usage that was removed in this PR.

Reply:
Yeah it was already unused, lost somewhere when I initially refactored this to the current state. That's why it's in an extra commit :)